ic-asset/src/operations.rs

```diff
@@ -59,6 +59,14 @@ pub(crate) fn create_new_assets(
             operations.push(BatchOperationKind::CreateAsset(CreateAssetArguments {
                 key: key.clone(),
                 content_type: project_asset.media_type.to_string(),
+                max_age: project_asset
+                    .asset_descriptor
+                    .config
+                    .cache
+                    .as_ref()
+                    .unwrap() // TODO
```

Comment:
I'm struggling to find a good and sleek pattern to turn Option<CacheConfig>, where CacheConfig.max_age == Option<u64> into Option<u64>, can you help @ericswanson-dfinity ?

Reply:
settled on 
```rust
            let max_age = project_asset
                .asset_descriptor
                .config
                .cache
                .as_ref()
                .and_then(|c| c.max_age)
                .map(|c| c);
```
sorry for spam